marker_ids: [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]
output_rel_path: /markers/
marker_pixel_size: 500
